crates/taplo-common/src/schema/associations.rs

```diff
@@ -256,9 +256,8 @@ impl<E: Environment> SchemaAssociations<E> {
 
     pub fn add_from_config(&self, config: &Config) {
         for rule in &config.rule {
-            let file_rule = match rule.file_rule.clone() {
-                Some(rule) => rule,
-                None => continue,
+            let Some(file_rule) = rule.file_rule.clone() else {
```

Comment:
I'm a bit afraid this forces recent compiler version (and slightly a "breaking" change regarding MSRV). Does clippy really suggests to do this?

Reply:
Seems to be https://rust-lang.github.io/rust-clippy/stable/index.html#option_if_let_else

Reply:
Hmm yeah, this let-else was stabilised in 1.65 rust which would be breaking for old versions.